android/src/main/java/com/reactbolt/sdk/RNBoltReactLibraryModule.java

```diff
@@ -311,7 +311,7 @@ private void validateExpiryDate(String expiryDate) throws ValidateException {
 
     @ReactMethod
     private void setupConsumerApiEndpoint(String url) {
-        CCConsumer.getInstance().getApi().setEndPoint(url);
+        CCConsumer.getInstance().getApi().setEndPoint("https://" + url);
```

Comment:
I assume that it's imperative this URL use HTTPS for security reasons.  Main thing I could see this affecting is local development, if the service can be run locally over HTTP.

Reply:
This was mainly to provide a consistent interface to JS because the iOS SDK adds the protocol automatically.  Since this points to a card connect service, I can't see a need to change it, unless they offer an image of their api for local testing at some point in the future.  Happy to reconsider if you think it's worth it.  I could add an optional protocol parameter, but it'd only be useful on Android.

Reply:
Makes sense— seems good to me!